src/components/VPagination/VPagination.ts

```diff
@@ -99,22 +104,22 @@ export default {
     },
     onResize () {
       const width = this.$el && this.$el.parentNode
-        ? this.$el.parentNode.clientWidth
+        ? (this.$el.parentNode as HTMLElement).clientWidth
```

Comment:
Does this need to be `parentNode`? `parentElement ` would work too without the type assertion.